backtester/backtester/models/trade.cpp

```diff
@@ -14,10 +14,11 @@ void trade_ns::to_json(json& j, const Trade& trade) {
 	case sell:
 		actionString = "sell";
 		break;
-	case nop:
+	case doNothing:
 		actionString = "nop";
```

Comment:
Just to be sure, did you mean to let the actionString at nop or did you forget to change it to doNothing?